spec/atlas/atlas_connectivity_spec.rb

```diff
@@ -11,13 +11,13 @@
 
   describe 'connection to Atlas' do
     it 'runs ismaster successfully' do
-      result = client.database.command(:ismaster => 1)
-      expect(result.documents.first['ismaster']).to be true
+      expect { client.database.command(:ismaster => 1) }
```

Comment:
```suggestion
      expect { client.database.command(:hello => 1) }
```

Reply:
`ismaster` is deprecated, we should use `hello` instead.

Reply:
I wasn't sure how closely we ought to mimic the other drivers; Go & Python are both still using `ismaster` here, but probably just because it's never been revisited. I'm okay switching to `hello`.